packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/attributeDenyCheck.js

```diff
@@ -0,0 +1,48 @@
+import operators from "@reactioncommerce/api-utils/operators.js";
```

Comment:
I am surprised it doesn't have it but this needs tests, as well as the other methods in the suite

Reply:
As discussed, tests are present in `filterShippingMethods.test.js`